app/assets/stylesheets/invitations.css.scss

```diff
@@ -5,11 +5,19 @@
 }
```

Comment:
my (possibly outdated) view on this issue is that a web-design should try to be responsive enough to fit relative size spezifications. there is no gain in aiming for pixel-perfect rendering for any specific output device as rendering will always have different "flavours" and you can never predict how changing display densities or zooming will behave.